src/assignmentone/Utils.java

```diff
@@ -0,0 +1,72 @@
+package assignmentone;
+
+/*
+ * This class is use to calculate tax of different items
+ */
+
+public final class Utils {
```

Comment:
for final class u should declare a private constructor.

Reply:
ok